config/manager/kustomization.yaml

```diff
@@ -5,6 +5,12 @@ generatorOptions:
   disableNameSuffixHash: true
 
 configMapGenerator:
-- name: manager-config
-  files:
+- files:
   - controller_manager_config.yaml
+  name: manager-config
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: bigkevmcd/cluster-bootstrap-controller
```

Comment:
Not sure what this image is referring to? 🤔

Reply:
An artifact of the build process, I reverted to the main version.